glide.yaml

```diff
@@ -16,6 +16,8 @@ import:
   version: ^0.3.0
 - package: github.com/vishvananda/netlink
 - package: github.com/Mellanox/rdmamap
+  vcs: git
+  version: master
```

Comment:
Can we avoid version locked to master? It's as good as not specifying a version at all.

Reply:
yes we should version it and update the commit

Reply:
@ahalim-intel updated with version v1.0.0 of rdmamap